charts/enterprise/clusterissuer/questions.yaml

```diff
@@ -306,7 +306,7 @@ questions:
                         default: ""
                     - variable: key
                       label: "Custom CA key (experimental)"
-                      description: "key Certiticate Authority"
+                      description: "CA Cert private key."
```

Comment:
please write CA in full

Reply:
Ive reverted questions.yaml as I'm no longer changing the user input to just take base64